src/System.Management.Automation/engine/hostifaces/PowerShellProcessInstance.cs

```diff
@@ -46,10 +46,17 @@ static PowerShellProcessInstance()
         /// <param name="credential"></param>
         /// <param name="initializationScript"></param>
         /// <param name="useWow64"></param>
-        public PowerShellProcessInstance(Version powerShellVersion, PSCredential credential, ScriptBlock initializationScript, bool useWow64)
+        /// <param name="workingDirectory"></param>
+        public PowerShellProcessInstance(Version powerShellVersion, PSCredential credential, ScriptBlock initializationScript, bool useWow64, string workingDirectory)
         {
             string processArguments = " -s -NoLogo -NoProfile";
 
+            if (workingDirectory != null)
```

Comment:
Should we also check for empty strings?

Reply:
fixed in 62a7b0c71ac60747f84ffba8f523ff0004fd17c9